woocommerce/wc-admin
# Fixed the Add First Product email note checks

Fixes #6231

This PR adds two extra checks to the `Add First Product` email note and also modifies the image.
The extra checks are:
* The note won't be sent to stores older than 30 days.
* The note won't be sent to stores with orders.

### Accessibility




### Screenshots
![screenshot-one wordpress test-2021 02 02-15_01_45](https://user-images.githubusercontent.com/1314156/106643072-f0beb900-6567-11eb-82e6-3a2ac9dc1438.png)


### Detailed test instructions:

- Checkout this branch.
- Create a zip for testing with `npm run zip:test`.
- Create a `jurassic.ninja` instance.
- Upload the plugin and activate it.
- Update the installation date (we need a store between 3 and 30 days old). You can do it with an SQL statement like this:

```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 5 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```

- Run the cron (this tool can help [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)).
- You should have received an email like the image above.
- Verify the note's status is `sent`. You can use an SQL statement like this:
```
SELECT `status` FROM `wp_wc_admin_notes` WHERE `name` = 'wc-admin-add-first-product-note'
```
- Now delete the note with an SQL statement like:
```
DELETE FROM `wp_wc_admin_notes` WHERE `name` = 'wc-admin-add-first-product-note';
```
- Add a new order and run the cron.
- No note should have been added.
- Remove the order, add a product and run the cron.
- No note should have been added.
- Delete the product and modify the store creation date to 32 days with an SQL statement like:
```
UPDATE `wp_options` SET `option_value`=UNIX_TIMESTAMP(DATE_SUB(NOW(), INTERVAL 32 day)) WHERE `option_name` = 'woocommerce_admin_install_timestamp';
```
- No note should have been added.
